roles/sat6repro/vars/main.sample.yml

```diff
@@ -11,19 +11,19 @@ rhelversion: 7
 satelliteversion: 6.1
 
 
-# Place the Satellite backup files only in satellite-clone/roles/sat6repro/files folder.
-# NOTE: Just type the file names here and don't include the path names.
+# Place the Satellite backup files only on the Destination node and not on the Ansible Control node.
+# NOTE: The backup files should be named exactly as mentioned below:
+#       1. config_files.tar.gz
+#       2. pgsql_data.tar.gz
+#       3. mongo_data.tar.gz
 perform_restore: true
```

Comment:
What does perform_restore mean? (asking from the user perspective, maybe we should add a comment here?)